app/assets/stylesheets/components/home_page_header.scss

```diff
@@ -1,25 +1,24 @@
-header {
-    padding-bottom: 60px;
+.hero {
+  text-align: center;
+  width: 100%;
 
-  h1 {
-    margin-bottom: 30px;
+  header {
+    padding-bottom: 3.75rem;
```

Comment:
why are we using rems for padding and margins?  IMO rems are best for font sizes because it's possible to change the base font-size site-wide and also possible to change it on your browser (for accessibility reasons!)

but I don' t really see a worthwhile reason to set things like padding and margins using rems because we do NOT want them to change if the user messes with their browser's font settings, or if we decide to change the site's main font size for some reason.